angular/projects/spark-angular/src/lib/components/sprk-tabbed-navigation/sprk-tabbed-navigation.component.ts

```diff
@@ -180,13 +175,27 @@ export class SprkTabbedNavigationComponent implements AfterContentInit {
       tabIDs = tabIDs.reverse();
       panelIDs = panelIDs.reverse();
 
-      this.panels.forEach(panel => {
+      this.panels.forEach((panel) => {
         panel.ref.nativeElement.setAttribute('id', panelIDs.pop());
         panel.ref.nativeElement.setAttribute('aria-labelledby', tabIDs.pop());
       });
     }
   }
 
+  /**
+   * @ignore
+   */
+  ngAfterContentInit(): void {
+    // Accounting for dynamic changes in the component
+    if (this.panels) {
```

Comment:
Is this check because its possible that someone would use the Tabs component without panels?

Reply:
Test By
- Delete check
- If error, remove all panels, see if it errors
- Report back WHY there's an error

Reply:
Theory: QueryList returns null

Reply:
This check is unecessary -- I removed it, removed all panels, and no errors.
I think it was spillover from stuff we were trying before. 
This is fixed in a commit that's about to be pushed!

Reply:
Spoke to soon, tests fail because panels doesn't exist
![image](https://user-images.githubusercontent.com/4342363/87819964-31a8e780-c83b-11ea-9371-59fd25bc9d96.png)

Reply:
Console.log panels and it shows a long list of panel Query Lists, and some of them are undefined

![image](https://user-images.githubusercontent.com/4342363/87820152-906e6100-c83b-11ea-8f6d-04bfad8d4711.png)

Reply:
Hmmm, it only fails in the tests?

Reply:
The issue was that we were calling ngOnContentInit manually outside of the lifecycle. I removed it, which made it only call normally, removing the need for the check.